cfgov/unprocessed/js/organisms/MegaMenuDesktop.js

```diff
@@ -46,7 +46,9 @@ function MegaMenuDesktop( menus ) {
        We'll use this later to check if we're still over the links,
        on mouse move. */
     const firstLevelMenus = _menus.getAllAtLevel( 1 );
-    _firstLevelDom = firstLevelMenus[0].data.getDom().container.parentNode;
+    if ( firstLevelMenus.length ) {
```

Comment:
Should we check the value of length here? Like `firstLevelMenus.length > 0`?

Reply:
I think we probably don't need to since 0 is falsy, unless we have a stylistic preference?

Reply:
Yeah I don't feel strongly on it, but unless it's a boolean value I'd probably explicitly show what value is being checked for.

Reply:
Updated in 69a10a6335428bdc5885c0e3917c71afbecbed78